contracts/tokens/SecurityToken.sol

```diff
@@ -781,4 +813,49 @@ contract SecurityToken is ERC20, ERC20Detailed, Ownable, ReentrancyGuard, Securi
         featureRegistry = PolymathRegistry(polymathRegistry).getAddress("FeatureRegistry");
         polyToken = PolymathRegistry(polymathRegistry).getAddress("PolyToken");
     }
+
+    //Ownable Functions
+
+    /**
+     * @return the address of the owner.
+     */
+    function owner() public view returns (address) {
+        return _owner;
+    }
+
+    /**
+     * @return true if `msg.sender` is the owner of the contract.
+     */
+    function isOwner() public view returns (bool) {
+        return msg.sender == _owner;
+    }
+
+    /**
+     * @dev Allows the current owner to relinquish control of the contract.
+     * @notice Renouncing to ownership will leave the contract without an owner.
+     * It will not be possible to call the functions with the `onlyOwner`
+     * modifier anymore.
+     */
+    function renounceOwnership() public onlyOwner {
```

Comment:
Agreed - removed. I guess this is in our current version as we inherit from Ownable which has this (unless this function isn't in the previous version of OZ),

Reply:
Yeah, It's part of the current ST. We missed it or didn't think that it will be an Issue. The recent acknowledgement change basically made me catch this lol. I would have probably not commented anything If we hadn't made that change. 

The PR is merged now so you can just use the functions from there and change the ack text.